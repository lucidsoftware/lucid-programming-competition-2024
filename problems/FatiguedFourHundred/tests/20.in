version https://git-lfs.github.com/spec/v1
oid sha256:be0d6d5c996e359c1185644a5f0804b7a855877730c36d770813d4c2ea7c60cc
size 482321
