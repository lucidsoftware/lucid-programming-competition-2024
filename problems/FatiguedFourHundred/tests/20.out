version https://git-lfs.github.com/spec/v1
oid sha256:924328e2262a9f047854938a68304eecc9c7a0d88507fa75693d3e05efa5eeaa
size 356362
